### PR TITLE
Change http to https for security links

### PR DESCRIPTION
**Pull request by huynq0911**
Please fill in the fields below to submit a pull request.  The more information
that is provided, the better.

**Fixes issue #**: N/A

**Description of the changes being introduced by the pull request**:
For security, we should change http into https links.

**Please verify and check that the pull request fulfills the following
requirements**:
N/A

### CODE REVIEW

File: docs/ADOPTERS.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Adopters
 * [Advanced Telematics Systems](https://advancedtelematic.com/en/press-releases/ats-is-integrating-the-uptane-security-framework-for-over-the-air-software-updates-to-connected-vehicles.html)
 * [Airbiquity](https://www.airbiquity.com/news/press-releases/airbiquity-unveils-new-otamatictm-release-further-strengthening-over-air-ota-software-and-data-management-offering-automotive)
-* [Automotive Grade Linux](http://docs.automotivelinux.org/docs/architecture/en/dev/reference/security/05-platform-security.html#agl-platform-software-update)
+* [Automotive Grade Linux](https://docs.automotivelinux.org/docs/architecture/en/dev/reference/security/05-platform-security.html#agl-platform-software-update)
```
Comment (lukpueh) [inline docs/ADOPTERS.md]: This link seems dead (at least the path).

File: docs/METADATA.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Metadata files provide information that clients can use to make update decisions
 
 The signed metadata files always include the time they were created and their expiration dates. This ensures that outdated metadata will be detected and that clients can refuse to accept metadata older than that which they've already seen.
 
-All TUF metadata uses a subset of the JSON object format. When calculating the digest of an object, we use the [Canonical JSON](http://wiki.laptop.org/go/Canonical_JSON) format. Implementation-level detail about the metadata can be found in the [spec](docs/tuf-spec.txt).
+All TUF metadata uses a subset of the JSON object format. When calculating the digest of an object, we use the [Canonical JSON](https://wiki.laptop.org/go/Canonical_JSON) format. Implementation-level detail about the metadata can be found in the [spec](docs/tuf-spec.txt).
```
Comment (lukpueh) [inline docs/METADATA.md]: This server does not support https.

File: tox.ini [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Tox (http://tox.testrun.org/) is a tool for running tests
+# Tox (https://tox.testrun.org/) is a tool for running tests
```
Comment (lukpueh) [inline tox.ini]: The subdomain link is dead and the organizational domain seems to be something else. Maybe this should point to https://tox.readthedocs.io/en/latest/ ?

File: docs/ADOPTERS.md (hunk 1 above)
Comment (huynq0911) [reply docs/ADOPTERS.md]: I have updated to new url : http://docs.automotivelinux.org/master/docs/architecture/en/dev/reference/security/README.html

File: docs/METADATA.md (hunk 2 above)
Comment (huynq0911) [reply docs/METADATA.md]: I have just reverted this to http

File: tox.ini (hunk 3 above)
Comment (huynq0911) [reply tox.ini]: Yes, I think so

**Comment by huynq0911** [comment]
> Thanks for the PR, @huynq0911. Have you tried any of the links? Some of them don't seem to work (see my comments inline). I vote for reverting this merge. We should seize the opportunity and go through the files in `docs` to update outdated links.

Thank for your review @lukpueh , I just updated by your comment and I create a new PR [#828](https://github.com/theupdateframework/tuf/pull/828) to update this. Please check it!

**Pull request merged**